scripts/scripts.js

```diff
@@ -158,6 +158,26 @@ function consolidateOfferBoxes(main) {
   }
 }
 
+/**
+ * get embed code for Wistia videos
+ *
+ * @param {*} url
+ * @returns
+ */
+export function embedWistia(url) {
```

Comment:
what about replacePlaceholder, autoplay options

Reply:
Hi, @jindaliiita can we approve this to get these blocks done?  This also does not need autoplay and placeholder, this is why I did not include them.

Reply:
If these are needed; the blocks can always be updated by the team that takes over; but all of the Wistia embeds on the site; don't have placeholders, or autoplay option.